src/App.vue

```diff
@@ -27,9 +27,9 @@ onMounted(async () => {
 </script>
 
 <template>
-  <div v-if="loadedConfig">
+  <template v-if="loadedConfig">
```

Comment:
out of curiosity, why are there nested templates?

Reply:
I just learned this today from this question, nice little trivia -> https://stackoverflow.com/questions/51995815/vuejs-template-vs-div-or-related-for-grouping-elements-for-conditional-rend